onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -74,16 +71,23 @@ class TensorrtExecutionProvider : public IExecutionProvider {
   std::shared_ptr<KernelRegistry> GetKernelRegistry() const override;
 
   void SetMaxBatchSize(const int batch_size) {
-      max_batch_size_ = batch_size;
+    max_batch_size_ = batch_size;
   }
 
   void SetMaxWorkspaceSize(const size_t workspace_size) {
-      max_workspace_size_ = workspace_size;
+    max_workspace_size_ = workspace_size;
   }
-    
+
+  std::unique_ptr<IndexedSubGraph> GetSubGraph(SubGraph_t graph_nodes_index, int& kernels_index,
```

Comment:
do all these methods here need to be public? (GetSubGraph, GetSupportedLst etc.)